opentelemetry/src/logs/mod.rs

```diff
@@ -30,6 +30,14 @@ pub enum LogError {
     #[error("Exporter timed out after {} seconds", .0.as_secs())]
     ExportTimedOut(Duration),
 
+    /// Processor is already shutdown
+    #[error("{0} already shutdown")]
```

Comment:
we are trying to get rid of this as part of https://github.com/open-telemetry/opentelemetry-rust/issues/1042 , so not sure why we need to add more to this?

Reply:
This will be moved to SDK eventually once global error handler is removed.